<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenderingMethodWidget</class>
 <widget class="QWidget" name="RenderingMethodWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="rendering_method_frame">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>421</width>
     <height>271</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="rendering_method_combobox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="russian_roulette_box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Russian roulette</string>
   </property>
   <widget class="QFrame" name="russian_roulette_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="ray_cast_epsilon_box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Ray cast epsilon</string>
   </property>
   <widget class="QDoubleSpinBox" name="ray_cast_epsilon_spinbox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="keyboardTracking">
     <bool>false</bool>
    </property>
    <property name="decimals">
     <number>6</number>
    </property>
    <property name="minimum">
     <double>0.000001000000000</double>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.000001000000000</double>
    </property>
    <property name="value">
     <double>0.000001000000000</double>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
